src/interface/storage.thrift

```diff
@@ -76,8 +91,8 @@ struct ResponseCommon {
 
 struct QueryResponse {
     1: required ResponseCommon result,
-    2: optional common.Schema vertex_schema,   // vertex related props
-    3: optional common.Schema edge_schema,     // edge related props
+    2: optional map<common.TagID, common.Schema>(cpp.template = "std::unordered_map")       vertex_schema,
+    3: optional map<common.EdgeType, common.Schema>(cpp.template = "std::unordered_map")    edge_schema,
```

Comment:
Use list instead,  the order is important for query engine.

Reply:
It's ok for me.  You'd better confirm it with @dutor ,  he has plan to change the logic about getting property by index instead of getting property by name in query engine.